cli_tools/gce_ovf_import/ovf_utils/ovf_programmatic_units_test.go

```diff
@@ -0,0 +1,63 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package ovfutils
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+)
+
+type getCapacityInGBTest struct {
+	expected        int
+	capacity        string
+	allocationUnits string
+	expectedErr     error
+}
+
+var getCapacityInGBTests = []getCapacityInGBTest{
+	// GB
+	{20, "20", "byte * 2^30", nil},
+	{20, "20", "byte * 2^30", nil},
+	{10, "10", "byte * 2^30", nil},
+	{1, "1", "byte * 2^30", nil},
+	{1024, "1024", "byte * 2^30", nil},
+	{5242880, "5242880", "byte * 2^30", nil},
+
+	// MB
+	{1, "1", "byte * 2^20", nil},
+	{1, "1024", "byte * 2^20", nil},
+	{5 * 1024, "5242880", "byte * 2^20", nil},
+
+	// TB
+	{1024, "1", "byte * 2^40", nil},
+	{5242880 * 1024, "5242880", "byte * 2^40", nil},
+
+	// Parse error due to allocation units not being recognized.
+	{0, "1024", "megabytes",
```

Comment:
What's the purpose of adding this test when we need to support "megabytes" anyway?

Reply:
It's a habit for working in unfamiliar code: introduce additional test coverage to help with my understanding and to increase overall coverage. The subsequent change should break this test, which is good, since it's exactly where we'll be modifying the code.